tests/core/helpers.py

```diff
@@ -1,7 +1,3 @@
-from eth_utils import decode_hex
-
-from evm.utils.padding import pad32
-from evm.vm.forks.sharding.transactions import ShardingTransaction
 
 
```

Comment:
Remove two empty lines.

Reply:
Flake8 fails on my machine if I do this (`tests/core/helpers.py:1:1: E302 expected 2 blank lines, found 0`). Pushed this anyway, let's see what Travis thinks about this.

Reply:
Apparently it's a known bug in pep8/pycodestyle: https://github.com/PyCQA/pycodestyle/issues/473

I've added a noqa for now.